eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -20,9 +20,27 @@ function Get-Suppression {
 
   # -NoEnumerate to prevent single-element arrays from being collapsed to a single object
   # -AsHashtable is closer to raw JSON than PSCustomObject
-  $suppressions = npx get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
+  $suppressions = npm exec --no -- get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
```

Comment:
Unrelated to crux of PR.  `npm exec --no --` is more predictable in scripts than `npx` and should always be used instead.

Reply:
The TypeSpec Requirements pipeline was not calling the npm install step so this was never working after your change on friday. And the last error check you added below started to cause the pipeline to fail which was good because we started to see that failure now.

I've added the npm install step to the pipeline in PR https://github.com/Azure/azure-rest-api-specs/pull/29008

Reply:
I expected `$suppressions = npm exec` to throw by default if the command returned a non-zero exit code.  I was surprised we need to manually check $LASTEXITCODE.